### PR TITLE
UnixFD: Move the support for Unix file descriptors to `dasbus.unix`

### PR DESCRIPTION
**Pull request by poncovka**
* Move all functions and classes related to the UnixFD support to `dasbus.unix`.
* Clean up the implementation of `GLibClientUnix` and `GLibServerUnix`.
* Clean up tests for swapping Unix file descriptors.

### CODE REVIEW

**Review by M4rtinK — APPROVED** [review]
Looks good to me, thanks! :)

**Review by VladimirSlavik — APPROVED** [review]
Looks good to me.

I suggest naming the module and things `unix_fd` or just `fd` - the "unix" bit is not exactly bringing any information. You already named the individual tests exactly that way, so it seems to be a sounds logic.

File: dasbus/unix.py [hunk 1]
```diff
@@ -210,35 +224,26 @@ class GLibServerUnix(GLibServer):
 
     @classmethod
     def set_call_reply(cls, invocation, out_type, out_value):
-        """Set the reply of the DBus call.
-
-        :param invocation: an invocation of a DBus call
-        :param out_type: a type of the reply
-        :param out_value: a value of the reply
-        """
+        """Set the reply of the DBus call."""
```
[the remark this answers is not in the document]
Comment (poncovka) [reply dasbus/unix.py:227]: Yes, I did. The arguments are same as in the parent.

**Comment by poncovka** [comment]
> I suggest naming the module and things `unix_fd` or just `fd` - the "unix" bit is not exactly bringing any information. You already named the individual tests exactly that way, so it seems to be a sounds logic.

I want to use the `dasbus.unix` namespace for all unix-specific code. In the future, there might be `dasbus.unix.fd`, but it is not necessary at this moment. This code will not work on non-unix systems, so I hope people will think twice before they will import anything from `dasbus.unix`.

**Pull request merged**